README.md

```diff
@@ -130,31 +131,34 @@ Example:
 }
 ```
 
-#### `digital_interrupts`
+#### `pins`
 
-[Interrupts](https://en.wikipedia.org/wiki/Interrupt) are a method of signaling precise state changes. Configuring digital interrupts to monitor GPIO pins on your board is useful when your application needs to know precisely when there is a change in GPIO value between high and low.
+Pins can be configured as GPIO pins and interrupts. [Interrupts](https://en.wikipedia.org/wiki/Interrupt) are a method of signaling precise state changes. Configuring digital interrupts to monitor GPIO pins on your board is useful when your application needs to know precisely when there is a change in GPIO value between high and low.
 
 - When an interrupt configured on your board processes a change in the state of the GPIO pin it is configured to monitor, it ticks to record the state change. You can stream these ticks with the board API’s [`StreamTicks()`](https://docs.viam.com/components/board/#streamticks), or get the current value of the digital interrupt with Value().
 - Calling [`GetGPIO()`](https://docs.viam.com/components/board/#getgpio) on a GPIO pin, which you can do without configuring interrupts, is useful when you want to know a pin’s value at specific points in your program, but is less precise and convenient than using an interrupt.
 
-Integrate `digital_interrupts` into your machine in the `attributes` of your board by following the **Config Builder** instructions, or by adding the following to your board’s JSON configuration:
+Integrate `pins` into your machine in the `attributes` of your board by following the **Config Builder** instructions, or by adding the following to your board’s JSON configuration:
 
 ```json
 // "attributes": { ... ,
-"digital_interrupts": [
+"pins": [
   {
     "name": "<your-digital-interrupt-name>",
-    "pin": "<pin-number>"
+    "pin": "<pin-number>",
+    "type": "<gpio/interrupt>"
   }
 ]
 ```
 
-The following properties are available for `digital_interrupts`:
+The following properties are available for `pins`:
 | Name | Type | Required? | Description |
 | ---- | ---- | --------- | ----------- |
 |`name` | string | **Required** | Your name for the digital interrupt. |
 |`pin`| string | **Required** | The pin number of the board's GPIO pin that you wish to configure the digital interrupt for. |
-|`type`| string | Optional | Default: `basic`: Tracks interrupt count. </li> </ul> |
+|`type`| string | **Required** | Whether the pin should be an `interrupt` or `gpio` pin |
```

Comment:
Is this required now? Okay if so but it should not be required in the future.

Reply:
nah its optional with a default to gpio, misinterpreted what i had before